src/Compilers/CSharp/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.cs

```diff
@@ -1016,6 +1016,75 @@ public class C
         #region Signing
 
         [Fact]
+        public void MaxSizeKey()
+        {
+            var pubKey = TestResources.General.snMaxSizePublicKeyString;
+            const string pubKeyToken = "1540923db30520b2";
```

Comment:
We have a simple pre-existing helper for calculating the bytes. In addition, the byte array is opaque, to me, but the string is readable and easily comparable with the output of sn.exe for debugging. The easiest way to compare the bytes was in the debugger anyway, so having it generated at runtime wasn't a significant difference.